geoutils/georaster/raster.py

```diff
@@ -526,7 +526,8 @@ def __str__(self) -> str:
         return self.info()
 
     def __eq__(self, other: object) -> bool:
-        """Check if a Raster's data and georeferencing is equal to another."""
+        """Check if a Raster's data (including unmasked values), mask, nodata, and georeferencing are all equal to
```

Comment:
You mean "including masked values"?
I would be exhaustive, so you're missing fill_value and dtype.